Nodejs/Product/Nodejs/SharedProject/ProjectNode.cs

```diff
@@ -590,7 +587,7 @@ public virtual string OutputBaseRelativePath
         /// <summary>
         /// Gets a collection of integer ids that maps to project item instances
         /// </summary>
-        internal HierarchyIdMap ItemIdMap => this.itemIdMap;
+        internal HierarchyIdMap ItemIdMap => HierarchyIdMap.Instance;
```

Comment:
>ItemIdMap [](start = 32, length = 9)

Seems like the callers could just use the constant directly, but no objection if you're minimizing the diff.